portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,15 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List; 
+import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Miah!</h1>");
+
+    // TODO:  Make ArrayList more abstract aka make it a List.
+    ArrayList<String> sampleContents = new ArrayList<String>();
```

Comment:
you should be able to make this change in this PR itself, just commit your changes from this branch and they should be on this PR :) (keeping in line with "quick fix" style comments should be addressed before merging the PR)